tests/src/Interop/NativeCallable/NativeCallableTest.csproj

```diff
@@ -37,5 +37,9 @@
   <ItemGroup>
     <Compile Include="NativeCallableTest.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <!-- This is needed to make sure native binary gets installed in the right location -->
+    <ProjectReference Include="CMakeLists.txt" />
```

Comment:
By giving the relative path to the cmake file in the project is as good as taking a project dependency on *.nativeproj in msbuild. There is back ground logic which translates the relative path to bring in the native files built by the project into the csproj's current directory